tests/components/peco/test_sensor.py

```diff
@@ -0,0 +1,256 @@
+"""Test the PECO Outage Counter sensors."""
+import asyncio
+
+from homeassistant.components.peco.const import DOMAIN
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+MOCK_ENTRY_DATA = {"county": "TOTAL"}
+COUNTY_ENTRY_DATA = {"county": "BUCKS"}
+INVALID_COUNTY_DATA = {"county": "INVALID"}
+
+
+async def test_sensor_available(
+    aioclient_mock: AiohttpClientMocker, hass: HomeAssistant
+) -> None:
+    """Test that the sensors are working."""
+    # Totals Test
+    aioclient_mock.get(
```

Comment:
Please consider patching the client library, instead of mocking the HTTP client.